wsgi.py

```diff
@@ -0,0 +1,6 @@
+from drift.app import create_app
```

Comment:
I tend to be confused by `wsgi.py` vs. `app.py` since wsgi just represents the interface vs. `app.py` being a clear entry point to me no matter your knowledge.

Reply:
updated to make filenames clearer. `app.py` is "the app" now, and `standalone_flask_server.py` is the nonstandard way